zips/zip-0234.md

```diff
@@ -16,138 +17,119 @@ License: BSD-2-Clause
 # Terminology
 
 The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”,
-and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+and “REQUIRED” in this document are to be interpreted as described in RFC 2119.
+[1]
 
 "Network upgrade" - to be interpreted as described in ZIP 200. [2]
 
-“Block Subsidy” - to be interpreted as described in the Zcash Protocol
-Specification (TODO ZIP Editors: link from comment).
+"Block Subsidy” - the algorithmic issuance of ZEC on block creation. Part of the
+consensus rules. Split between the miner and the Dev Fund. Also known as Block
+Reward.
 
-“Issuance” - the sum of Block Subsidies over time. (TODO ZIP Editors: work out
-if this definition is correct or can be removed).
-
-“`ZsfBalanceAfter(h)`” is the total ZEC available in the Zcash Sustainability
-Fund (ZSF) after the transactions in block `h`, described in ZIP draft-zsf.md.
-In this ZIP, the Sustainability Fund is used to pay out Block Subsidies from
-unmined ZEC, and other fund deposits.
+"Issuance" - The method by which ZEC becomes available for circulation on the
+network.
 
 Let `PostBlossomHalvingInterval` be as defined in [#protocol-diffadjustment]_.
 
+"Money Reserve" - `MAX_MONEY - ChainValue`, where `ChainValue` includes all ZEC
+available on the network, across all value pools.
 
 # Abstract
 
 This ZIP proposes a change to how nodes calculate the block subsidy.
 
 Instead of following a step function around the 4-year halving intervals
-inherited from Bitcoin, we propose a slow exponential “smoothing” of the curve.
-The new issuance scheme would approximate the current issuance over 4-year
-intervals.
+inherited from Bitcoin, we propose a smooth logarithmic curve, defined as a
+fixed portion of the current value of the Money Reserve at a given block height.
 
-This ZIP depends on the ZIP introducing the Zcash Sustainability Fund
-(ZIP-XXX).
+The new issuance scheme would approximate the current issuance over 4-year
+intervals, assuming no ZEC is burned during that time, and retains the overall
+supply cap of `MAX_MONEY`.
 
 # Motivation
 
+Key Objectives:
+
+1. We want to introduce an automated mechanism that allows users of the network
+   to contribute to the long-term sustainability of the network.
+2. We want to enable ZEC that has been burned to be recreated in the future to
+   benefit network sustainability.
+3. We want to retain the existing ZEC supply cap of 21 million.
+4. We want the issuance rate to remain similar to the historical rate for Zcash
+   (and before that, Bitcoin).
+5. We want issuance to be easy for all network users to understand and predict.
```

Comment:
There has been a relatively large amount of discussion about activation height and the impacts of different choices. We propose to choose the activation height that has the minimal short term change from the current issuance schedule with this new Key Objective 6:

```suggestion
5. We want issuance to be easy for all network users to understand and predict.
6. We want the new issuance to activate at a block with as minimal a delta from the current issuance as possible.
```

Reply:
@shielded-nate as a consequence of the fact that this branch belongs to the `ShieldedLabs` organization, I'm not able to directly apply these changes. Could you please apply these changes, and also rebase this PR to remove the rendering artifacts? Having the rendering in this PR causes unnecessary conflicts; the ZIP  Editors will re-render everything once all the stabilized NU7 ZIPs have been merged.